README.md

```diff
@@ -1 +1,71 @@
-# capstone-1
\ No newline at end of file
+
+# Project Name
+AI conference, castone-1
+
+## Live Demo
+visit me here:
+
+## Built With
+- html
+- css
+- node
+- git
+- bootstrap
+- VS Code
+- JavaScript
+
+## Link to video
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+install git, node, any code editor in your local machine
+
+### Setup
+
+Step 1: Type the following command into a git shell
+
+git clone https://github.com/tamana-Balkhi/capstone-1.git
+
+### Install
+
+Run the following commands to install dependencies
+
+```
+
+npm install --save-dev hint@7.x
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+
+```
+### Usage
+open index.html in your local browser
+run these commonds to avoid linters errors```
+
+npx hint .
+npx stylelint "**/*.{css,scss}"
+npx eslint --fix .
+
+## Authors
+👤 tamana balkhi
+
+- GitHub: [Tamana Balkhi](https://github.com/tamana-Balkhi)
+- Twitter: [Tamana Balkhi](https://twitter.com/Tamana1377)
+- Gmail: tamana.balkhi1377@gmail.com
+
+##  Design Credit
+[Original design idea by Cindy Shin in Behance](https://creativecommons.org/licenses/by-nc/4.0/)

```

Comment:
Great! You are giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) ✔